alerts/utilization.libsonnet

```diff
@@ -14,8 +14,8 @@
               severity: 'warning',
             },
             annotations: {
-              message: 'Storage cluster is nearing full. An expansion is required',
-              description: 'The utilization of storage cluster has crossed 85%.',
+              message: 'Storage cluster is nearing full. Expansion is required.',
+              description: 'Storage cluster utilization has crossed 85%.',
```

Comment:
we can add "Storage cluster expansion is required"

Reply:
Same as above